travis/minimesosFile

```diff
@@ -55,7 +55,7 @@ minimesos {
         imageName = "containersol/mesos-agent"
         imageTag = "1.0.0-0.1.0"
         loggingLevel = "# INHERIT FROM CLUSTER"
-        portNumber = 5052
+        portNumber = 5051
```

Comment:
The travis changes are required for build, which is working.

I tested the changes on the scheduler minimesos file to run integration tests locally, they worked.